core/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
     api 'com.google.android.filament:gltfio-android:1.9.25'
     implementation files("libs/libsceneform_runtime_schemas.jar")
 
-    api "com.google.ar:core:1.23.0"
+    api "com.google.ar:core:1.24.0"
```

Comment:
Did you see any issue with the 1.24.0 ?

Reply:
Not so far, I'm using 1.24 now since the release of this version.